### ISSUE TITLE
[Screen reader-Howdy]: Heading tag h1 is not defined for howdy dashboard page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#22]
**User Experience**
Users who rely on screen reader will get effected to navigate the elements if the heading tag h1 is not defined for howdy dashboard page .then the user will be affected to navigate to the headings mode.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.379)
URL: https://hcl-wxe3660.howdy.ai/dashboard 
Browser: Edge
 
**Repro Steps**
Step 1: Open the https://hcl-wxe3660.howdy.ai/dashboard  url in Edge browser
Step 2: "Howdy Dashboard" page gets opened.
Step 3: Open the developer tool by using F12 key  and verify the heading tag h1

Step 4: Observe the issue.

**Actual Result**
Heading tag h1 is not defined for howdy dashboard page.
 
**Expected Result**
Heading tag h1 should be defined for howdy dashboard page.

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}
![MAS2 4 6_Howdy](https://user-images.githubusercontent.com/48272005/54522429-359ccf00-4993-11e9-89af-e63347954bdb.png)

**Comment by benbrown** [comment]
* h3 replaced by an h1

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
